Tests/ExtractionTests/ExtractionTests.swift

```diff
@@ -1,168 +1,195 @@
 
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-import XCTest
+import Testing
 import Foundation
 
 @testable import EvolutionMetadataModel
 @testable import EvolutionMetadataExtraction
 
-final class ExtractionTests: XCTestCase {
-    
-    func urlForSnapshot(named snapshotName: String) throws -> URL {
-        try XCTUnwrap(Bundle.module.url(forResource: snapshotName, withExtension: "evosnapshot", subdirectory: "Resources"), "Unable to find snapshot \(snapshotName).evosnapshot in test bundle resources.")
-    }
-    
-    func data(forResource name: String, withExtension ext: String) throws -> Data {
-        let url = try XCTUnwrap(Bundle.module.url(forResource: name, withExtension: ext, subdirectory: "Resources"), "Unable to find resource \(name).\(ext) in test bundle resources.")
-        let data = try XCTUnwrap(Data(contentsOf: url), "Unable to read data from \(name).\(ext)")
-        return data
-    }
-    
-    func string(forResource name: String, withExtension ext: String) throws -> String {
-        let data = try data(forResource: name, withExtension: ext)
-        let string = try XCTUnwrap(String(data: data, encoding: .utf8), "Unable to make string from contents of \(name).\(ext)")
-        return string
-    }
-    
-    // Convenience to write expected and actual metadata files to disk for comparison in a diff tool
-    func writeJSONFilesToPath(expected: Data, actual: Data, path: String, prefix: String? = nil) throws {
-        let filePrefix: String
-        if let prefix { filePrefix = "\(prefix)-" } else { filePrefix = "" }
-        try expected.write(to: FileUtilities.expandedAndStandardizedURL(for: path).appending(path: "\(filePrefix)expected.json"))
-        try actual.write(to: FileUtilities.expandedAndStandardizedURL(for: path).appending(path: "\(filePrefix)actual.json"))
-    }
+struct `Extraction tests` {
```

Comment:
Would prefer suite names in title case, so it reads more like a header in reports.

I realize suites are implied by a type containing tests, but is there a drawback to explicitly marking with `@Suite` on the line above?

```suggestion
@Suite
struct `Extraction Tests` {
```

Reply:
> I realize suites are implied by a type containing tests, but is there a drawback to explicitly marking with `@Suite` on the line above?

No, there's no technical drawback. It's slightly more verbose but I don't mind adding it.